demo/app/android/app/src/main/kotlin/walletsdk/flutter/app/MainActivity.kt

```diff
@@ -57,10 +58,16 @@ class MainActivity : FlutterActivity() {
                                 result.success(userPinRequired)
 
                             } catch (e: Exception) {
+                                val err = Walleterror.parse(e.message)
```

Comment:
Is it possible to pass a struct through to the exception instead of having to parse the message string? (no idea if this is possible - just wanted to check and see if gomobile/the bindings allow it)

Reply:
I don't find way.